metadata.rb

```diff
@@ -1,21 +1,21 @@
-name 'chef-splunk'
-maintainer 'Chef Software, Inc.'
-maintainer_email 'cookbooks@chef.io'
-license 'Apache-2.0'
-description 'Manage Splunk Enterprise or Splunk Universal Forwarder'
-version '7.1.0'
+name              'chef-splunk'
+maintainer        'Sous Chefs'
+maintainer_email  'help@sous-chefs.org'
+license           'Apache-2.0'
+description       'Manage Splunk Enterprise or Splunk Universal Forwarder'
+version           '7.1.0'
+source_url        'https://github.com/sous-chefs/chef-splunk'
+issues_url        'https://github.com/sous-chefs/chef-splunk/issues'
+chef_version      '>= 13.11'
 
-supports 'debian', '>= 8.9'
-supports 'ubuntu', '>= 16.04'
-supports 'redhat', '>= 6.9'
-supports 'centos', '>= 6.9'
 supports 'amazon'
+supports 'centos'
+supports 'debian'
+supports 'redhat'
+supports 'ubuntu'
 
 # please read the README.md section regarding data bag fallback if you
 # do not use chef-vault
+depends 'apt'
 depends 'chef-vault', '~> 4.0'
 depends 'ec2-tags-ohai-plugin', '~> 0.2.4'
```

Comment:
I think this is a better way around this and frees up the dependency to folks.
```suggestion
depends 'chef-vault', '>= 4.0'
depends 'ec2-tags-ohai-plugin', '>= 0.2.4'
```